kajas/src/app/login/login.component.ts

```diff
@@ -0,0 +1,123 @@
+  modalMessage: string = '';
+
+  constructor(
+    private renderer: Renderer2,
+    private fb: FormBuilder,
+    private http: HttpClient,
+    private router: Router,
+    private sessionStorage: SessionStorageService
+  ) {
+    this.loginForm = this.fb.group({
+      email: ['', [Validators.required, Validators.email]],
+      password: ['', [Validators.required, Validators.minLength(6)]],
+    });
+  }
+
+  ngOnInit(): void {
+    this.setInitialStyles();
+    window.addEventListener('resize', this.applyBackground.bind(this));
+  }
+
+  ngOnDestroy(): void {
+    this.revertStyles();
+    window.removeEventListener('resize', this.applyBackground.bind(this));
+  }
+
+  private setInitialStyles(): void {
+    this.renderer.setStyle(document.body, 'height', '100%');
+    this.renderer.setStyle(document.body, 'overflow', 'hidden');
+    this.renderer.setStyle(document.body, 'display', 'flex');
+    this.renderer.setStyle(document.body, 'justify-content', 'center');
+    this.renderer.setStyle(document.body, 'align-items', 'center');
+    this.renderer.setStyle(
+      document.body,
+      'background',
+      'url("../../assets/login_bg.png") center/cover no-repeat'
+    );
+
+    this.renderer.setStyle(document.documentElement, 'height', '100%');
+    this.renderer.setStyle(document.documentElement, 'overflow', 'hidden');
+    this.applyBackground();
+  }
+
+  private revertStyles(): void {
+    this.renderer.removeStyle(document.body, 'height');
+    this.renderer.removeStyle(document.body, 'overflow');
+    this.renderer.removeStyle(document.body, 'display');
+    this.renderer.removeStyle(document.body, 'justify-content');
+    this.renderer.removeStyle(document.body, 'align-items');
+    this.renderer.removeStyle(document.body, 'background');
+
+    this.renderer.removeStyle(document.documentElement, 'height');
+    this.renderer.removeStyle(document.documentElement, 'overflow');
+  }
+
+  private applyBackground(): void {
+    if (window.innerWidth <= 425) {
+      this.renderer.setStyle(
+        document.body,
+        'background',
+        'url("../../assets/login_mbg.png") center/cover no-repeat'
+      );
+    } else {
+      this.renderer.setStyle(
+        document.body,
+        'background',
+        'url("../../assets/login_bg.png") center/cover no-repeat'
+      );
+    }
+  }
+
+  onSubmit(): void {
+    if (this.loginForm.valid) {
+      this.http
+        .post('http://localhost:4000/api/login', this.loginForm.value)
+        .subscribe(
+          (response: any) => {
+            this.modalMessage = 'Login Success! Welcome to Kajas!';
+            this.showModal = true;
+            console.log(response.user.user_id);
+            this.sessionStorage.set('id', response.user.user_id);
+            this.sessionStorage.set('username', response.user.username);
+            this.sessionStorage.set('email', response.user.email);
+
+            setTimeout(() => {
+              console.log('Navigating to /signup');
```

Comment:
### Remove console.logs() and unneccessary codes in the production environment.
- Use console.error() for logging error messages.
- Remove codes/variables that are not used.
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g2637aafa621_0_0